class/class3.1/src/Helpers.cpp

```diff
@@ -0,0 +1,18 @@
+#include "Helpers.hpp"
+
+double dummyFunc(double x) {
+    return x > 0 ? x : 0;
+}
+
+std::string stringRepeater(const std::vector<int>& numbers, const std::string& line) {
+    if (numbers.empty() || line == "") return "";
+
+    std::string result = "";
+    auto biggestNum = std::max_element(numbers.begin(), numbers.end());
```

Comment:
std::ranges::max(numbers)
and with ranges we can use repeat and so on to make this implementation even shorter something like
```
auto repeatCount = vec.empty() ? 0 : std::ranges::max(vec);
return repeatCount > 0 ?
            ? std::views::repeat(str, repeatCount) | std::views::join | std::ranges::to<std::string>()
            : "";
```